src/lib/y2partitioner/yast_nfs_client.rb

```diff
@@ -230,6 +233,7 @@ def newbut_handler(legacy)
       log.info "Test mount failed, so removing NFS from current graph: #{nfs.inspect}"
       current_graph.remove_nfs(nfs)
       set_client_list
+      call_client("RefreshUI")
```

Comment:
Due to the cyclic dependency commented above, we are going to move this call to the NfsClient4partClient itself, just at the end of [FromStorage](https://github.com/yast/yast-nfs-client/blob/1e2fadaaf16f2d2eddd87065fdfed3f05785fc7a/src/clients/nfs-client4part.rb#L38) action. If it works, this PR can be discarded and the fix will live only in yast-nfs-client.